kustomize/components/network-policy/README.md

```diff
@@ -1,28 +1,39 @@
-# NetworkPolicy + OnlineBoutique
+# NetworkPolicy
 
-This guide contains instructions for configuring `NetworkPolicy` resources for the OnlineBoutique apps by leveraging the [GKE Dataplane V2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2) feature.
+This guide contains instructions for deploying Online Boutique with [`NetworkPolicy`](https://kubernetes.io/docs/concepts/services-networking/network-policies/) resources.
 
-## Steps
- 
-1. You need to have a GKE cluster created with [GKE Dataplane V2 enabled](https://cloud.google.com/kubernetes-engine/docs/how-to/dataplane-v2).
+To use `NetworkPolicy` objects in Google Kubernetes Engine (GKE), you will need a GKE cluster with network policy enforcement enabled.
+To learn more, see the official [GKE documentation on enabling network policy enforcement](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy#enabling_network_policy_enforcement).
+You do not need to enable network policy enforcement in clusters that use [GKE Dataplane V2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2).
 
-As an example, here is the option you need to use when creating your cluster:
-```sh
-gcloud container clusters create \
-    --enable-dataplane-v2
```

Comment:
I've removed the step:
```
gcloud container clusters create \
    --enable-dataplane-v2
```
... because these GKE-specific instructions [documented in the GKE official docs](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy#enabling_network_policy_enforcement). We should link to those docs are much as possible — so we're relying on TWs' work as much as possible. :)

Reply:
(I'm open to bringing it back.)